docs/gathered-data.md

```diff
@@ -524,6 +525,7 @@ Gathered metrics:
   - "namespace:container_memory_usage_bytes:sum": 4.5+
   - "virt_platform metric": 4.6.34+, 4.7.16+, 4.8+
   - "vsphere_node_hw_version_total": 4.7.11+, 4.8+
+  - "console_helm_installs_total": 4.10+
```

Comment:
4.10 GA is out so it should be 4.11+ (we should be able to make it)

Reply:
Ok so it cannot be backtracked is it..Like the operator will gather only the metrics for 4.11 ocp version onwards.

Reply:
It can be backported (we can discuss it once it's merged in master).

Reply:
Made the change for console_helm_installs_total ocp version to 4.11+